iothub/service/src/IotHubServiceClientOptions.cs

```diff
@@ -101,5 +103,28 @@ public class IotHubServiceClientOptions
         /// </para>
         /// </remarks>
         public TimeSpan AmqpConnectionKeepAlive { get; set; } = TimeSpan.FromMinutes(2);
+
+        /// <summary>
+        /// A callback for remote certificate validation.
+        /// </summary>
+        /// <remarks>
+        /// If incorrectly implemented, your device may fail to connect to IoT Hub and/or be open to security vulnerabilities.
+        /// <para>
+        /// This feature is only applicable for HTTP connections and for AMQP TCP connections. AMQP web socket communication
+        /// does not support this feature.
+        /// </para>
+        /// </remarks>
+        public RemoteCertificateValidationCallback RemoteCertificateValidationCallback { get; set; } = DefaultRemoteCertificateValidation;
```

Comment:
Does this apply to both device and module client?

Reply:
Yes, but MQTT also supports this in device/module client